resilience4j-spring-boot-common/src/main/java/io/github/resilience4j/circuitbreaker/autoconfigure/AbstractCircuitBreakerConfigurationOnMissingBean.java

```diff
@@ -81,24 +81,30 @@ public CircuitBreakerRegistry circuitBreakerRegistry(
     @ConditionalOnMissingBean
     @Conditional(value = {AspectJOnClasspathCondition.class})
     public CircuitBreakerAspect circuitBreakerAspect(CircuitBreakerRegistry circuitBreakerRegistry,
-        @Autowired(required = false) List<CircuitBreakerAspectExt> circuitBreakerAspectExtList,
+        @Autowired(required = false) List<CircuitBreakerDecoratorExt> circuitBreakerAspectExtList,
```

Comment:
Could you change variable name? 
```suggestion
        @Autowired(required = false) List<CircuitBreakerDecoratorExt> circuitBreakerDecoratorExtList,
```